trusted-committer/03-keeping-the-community-healthy.asciidoc

```diff
@@ -1,80 +1,76 @@
+code and code health only. To ensure success in the long run, Trusted Committers should
+strive to keep the community building the software healthy
+as well. Because of that, they must strike a good balance between ensuring product quality and growing a healthy community.
 
 What does a healthy community look like? Quite simply, in a healthy community,
-https://github.com/InnerSourceCommons/InnerSourceLearningPath/blob/master/contributor/01-introduction-article.asciidoc[_Contributors_] tend to stick around, can spend most of their time on developing
-software and are able to level up their abilities. As a result, a healthy
-community will also be growing.
+https://github.com/InnerSourceCommons/InnerSourceLearningPath/blob/master/contributor/01-introduction-article.asciidoc[_Contributors_] tend to stick around, can spend most of their time developing software, and are able to level up their abilities.
+As a result, a healthy community will be continually growing.
 
 Why do https://github.com/InnerSourceCommons/InnerSourceLearningPath/blob/master/contributor/01-introduction-article.asciidoc[_Contributors_] join and stick around in a community? Some do because they
 subscribe to the purpose or the mission of the community. It is the Trusted Committer's job to
 clearly articulate and promote this purpose. The importance of this is often
-not recognized - marketing a community and their products is truly essential.
+not recognized, but marketing a community and its products is truly essential.
 
-Another, more obvious reason for people to stick around is that they enjoy
-working with other members of the community, including the Trusted Committers. In my
-experience, what this comes down to is that community members treat and
-communicate with each other with utmost respect. Contributions are treated as
-like a gift or a donation, rather than something that detracts from their own
-work and they laud excellent and especially first contributions. The Trusted Committer's job
-in all this is primarily to set an example for others, similarly to setting an
-example for the level of software quality that is expected.  If necessary, the
-Trusted Committers are the ones who should create and enact a code of conduct for the
-community. Should there be community members whose behavior is detrimental or
-even toxic to the community health, it is the Trusted Committer's responsibility to either
-try and change or contain this or, in the worst case, to remove people from the
-community. Trusted Committers should create opportunities for people to get together
-regularly and get to know each other personally.
+Another, more obvious reason for people to stick around is that they
+enjoy working with other members of the community, including the Trusted Committers. A thriving community is one where members treat
+and communicate with each other with the utmost respect. Contributions are
+treated like gifts or donations, and excellent (especially
```

Comment:
I liked the mention of "rather than something that detracts from their own work" - this is something I have repeatedly seen leading to bulk rate handling of the contribution and hence more often than not failure of the relationship / attempt to contribute, especially on first contributions where there's no pre-established rapport.
 
I'm open for a different phrasing, but giving this difference is valuable in my opinion.
Academic publications call this "prioritized mentorship" if I remember correctly and list it as one of the pillars of working InnerSource..

Reply:
> Contributions are treated as gifts or donations, rather than something that detracts from `their` own work...

It's not very clear in this case who "they" are, as TCs are last mentioned two sentences prior.
@lenucksi - will this version work, in your opinion?

> Contributions are treated like gifts or donations `rather than distractions`, and excellent (especially first) contributions are lauded.  

Please let me know, I will adjust accordingly.

Reply:
Can you add your suggestions as [commit suggestions](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/commenting-on-a-pull-request)? That way it is easier to see how they will work. They are available via the +/- symbol in the toolbar here, i.e. the first icon.